### PR TITLE
OKTA-449821: Splunk cloud log streaming dev docs

### PR DESCRIPTION
**Pull request by tomaszwrobel-okta**
## Description:
- Adding Splunk Cloud Log Streaming dev docs for EA
- **Is this PR related to a Monolith release?** Yes, 2022.04.0 
⚠️ Release notes for log streaming is handled by Data Analytics team

### Resolves:

* [OKTA-449821](https://oktainc.atlassian.net/browse/OKTA-449821)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/reference/api/log-streaming/index.md [hunk 1]
```diff
@@ -653,3 +676,25 @@ The AWS EventBridge Settings object specifies the configuration for the `aws_eve
...
+  "host": "acme.splunkcloud.com"
+  "token": "11111111-1111-2222-2222-222222222222"
+}
+```
+
+#### Splunk Cloud Settings properties
+
+| Property      | Description                                                  | DataType                                                       | Nullable | Unique | Readonly | MinLength | MaxLength |
+| ------------- | ------------------------------------------------------------ | -------------------------------------------------------------- | -------- | ------ | -------- | --------- | --------- |
+| host            | The domain for your Splunk Cloud instance without http or https. For example: `acme.splunkcloud.com`                                       | String                                                         | FALSE    | FALSE   | FALSE     |      17     |     116      |
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/reference/api/log-streaming/index.md:697]:
```suggestion
| host            | The domain name for your Splunk Cloud instance. Don't include `http` or `https` in the string. For example: `acme.splunkcloud.com`                                       | String                                                         | FALSE    | FALSE   | FALSE     |      17     |     116      |
```

File: packages/@okta/vuepress-site/docs/reference/api/log-streaming/index.md [hunk 2]
```diff
@@ -653,3 +676,25 @@ The AWS EventBridge Settings object specifies the configuration for the `aws_eve
...
+| token     | The HEC token for your Splunk Cloud HTTP Event Collector.              | String (GUID format)  | FALSE | FALSE | FALSE  |  36 |  36   |
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/reference/api/log-streaming/index.md:698]:
```suggestion
| token     | The HEC token for your Splunk Cloud HTTP Event Collector             | String (GUID format)  | FALSE | FALSE | FALSE  |  36 |  36   |
```

File: packages/@okta/vuepress-site/docs/reference/api/schemas/index.md [hunk 3]
```diff
@@ -1551,7 +1551,7 @@ The following response is only a subset of properties for brevity.
 
 <ApiOperation method="get" url="/api/v1/meta/schemas/logStream/${typeId}" />
 
-Fetches the schema for a Log Stream type. The `${typeId}` element in the URL specifies the Log Stream type, which can be the `aws_eventbridge` literal to retrieve the AWS EventBridge type schema.
+Fetches the schema for a Log Stream type. The `${typeId}` element in the URL specifies the Log Stream type, which can be the `aws_eventbridge`, literal to retrieve the AWS EventBridge type schema or the `splunk_cloud_logstreaming`, literal to retrieve the Splunk Cloud type schema.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/reference/api/schemas/index.md:1554]:
```suggestion
Fetches the schema for a Log Stream type. The `${typeId}` element in the URL specifies the Log Stream type, which is either `aws_eventbridge` or `splunk_cloud_logstreaming`. Use the `aws_eventbridge` literal to retrieve the AWS EventBridge type schema, and use the `splunk_cloud_logstreaming` literal retrieve the Splunk Cloud type schema.
```

**Review by vanngo-okta — APPROVED** [review]
A couple of small nits, but otherwise, looks good! 🚀

File: packages/@okta/vuepress-site/docs/reference/api/log-streaming/index.md [hunk 4]
```diff
@@ -228,6 +228,28 @@ curl -v -X GET \
...
+    "lastUpdated": "2021-11-30T15:16:09.000Z",
+    "created": "2021-11-30T15:16:09.000Z",
+    "status": "ACTIVE",
+    "settings": {
+      "host": "acme.splunkcloud.com",
+      "token": "1e652bb8-3ef8-427b-9f00-222e1bbe3832"
+    },
+    "_links": {
+      "self": {
+        "href": "http://${yourOktaDomain}/api/v1/logStreams/0oa129gzYyV6oWs1D0g4",
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/log-streaming/index.md:245]:
```suggestion
        "href": "http://{yourOktaDomain}/api/v1/logStreams/0oa129gzYyV6oWs1D0g4",
```

File: packages/@okta/vuepress-site/docs/reference/api/schemas/index.md [hunk 5]
```diff
@@ -1830,6 +1830,59 @@ For brevity, the following response doesn't include all available properties.
         "name": "Name can't exceed 100 characters."
       }
     }
+  },
+  {
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "http://${yourOktaDomain}/api/v1/meta/schemas/logStream/splunk_cloud_logstreaming",
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/schemas/index.md:1836]:
```suggestion
    "$id": "http://{yourOktaDomain}/api/v1/meta/schemas/logStream/splunk_cloud_logstreaming",
```

**Review by susanharper-okta — APPROVED** [review]
Three $ found with variables in the responses. We were just talking about this! :) Approving to not hold you up.

**Pull request merged**